src/features/form/layout/formLayoutSlice.ts

```diff
@@ -114,6 +115,8 @@ const formLayoutSlice = createSagaSlice((mkAction: MkActionType<ILayoutState>) =
         const { settings } = action.payload;
         if (settings && settings.pages) {
           updateCommonPageSettings(state, settings.pages);
+          const receiptLayoutName = settings.receiptLayoutName;
+          state.uiConfig.receiptLayoutName = receiptLayoutName;
```

Comment:
Småpirk, men hvis `receiptLayoutName` ligger i `settings` og ikke på `settings.pages` så burde vel den settes utenfor `if`-en, i tillegg er kanskje `const`-en litt overflødig.

```suggestion
        const { settings } = action.payload;
        state.uiConfig.receiptLayoutName = settings?.receiptLayoutName;
        if (settings && settings.pages) {
          updateCommonPageSettings(state, settings.pages);
```